src/components/containers/PlotlyFold.js

```diff
@@ -149,13 +154,37 @@ Fold.propTypes = {
   messageIfEmpty: PropTypes.string,
   localize: PropTypes.func,
   name: PropTypes.string,
+  forceVisibility: PropTypes.bool,
 };
 
-Fold.contextTypes = Object.assign(
+PlotlyFold.contextTypes = Object.assign(
   {
     deleteContainer: PropTypes.func,
   },
   containerConnectedContextTypes
 );
 
-export default localize(Fold);
+export default localize(PlotlyFold);
+
+export class Fold extends PlotlyFold {}
+
+Fold.plotly_editor_traits = {
+  foldable: true,
+};
+
+Fold.defaultProps = {
+  forceVisibility: true,
+};
+
+Fold.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  folded: PropTypes.bool,
+  toggleFold: PropTypes.func,
+  hideHeader: PropTypes.bool,
```

Comment:
I removed the isRequired here...
BUT, I am testing in my tests that those props are there for `<Fold/>`s
I don't know why those console.errors were occurring..